python/tidy/tidy.py

```diff
@@ -42,8 +43,33 @@
 sys.path.append(os.path.join(WPT_PATH, "tests"))
 sys.path.append(os.path.join(WPT_PATH, "tests", "tools", "wptrunner"))
 
-# Default configs
-config = {
+TCF = TypeVar("TCF")
```

Comment:
It is not clear to me why this type variable is called `TCF`. Maybe we can use a more descriptive name here.